demisto_sdk/commands/validate/validators/BC_validators/BC107_is_valid_toversion_on_modified.py

```diff
@@ -14,30 +16,115 @@
 
 class IsValidToversionOnModifiedValidator(BaseValidator[ContentTypes]):
     error_code = "BC107"
-    description = (
-        "Check that the toversion property was not changed on existing Content files."
-    )
-    rationale = "Changing the `toversion` field for a content item can break backward compatibility."
-    error_message = "Changing the maximal supported version field `toversion` is not allowed. Please undo, or request a force merge."
+    description = "Check that the toversion property is valid."
+    rationale = "Changing the `toversion` field for a content item should include adding a new item to replace it."
+    error_message = "{0}"
```

Comment:
Let's not remove the message.
We should remember that the main purpose of the validation is to ensure no changes has been done to the toversion field.
However, the new issue is to allow these changes in very specific cases.
Therefore the error should be something like this:
"Changing the maximal supported version field `toversion` is not allowed, unless you're adding new content_item with the same id <content_item_id> and their from/to version fulfills the following: <valid_cases>